namespaces/cloud-platform-live-0.k8s.integration.dsd.io/kubewatch/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kubewatch-admin
+  namespace: kubewatch
+subjects:
+  - kind: Group
+    name: "github:apply-for-legalaid"
```

Comment:
**NOTE:** Not sure if this is correct. This service could be generic across the MOJ?

Reply:
@lostie This is correct as you will be responsible for kubewatch in your namespace